applications/serial_lte_modem/src/http_c/slm_at_httpc.c

```diff
@@ -504,6 +505,7 @@ static void httpc_thread_fn(void *arg1, void *arg2, void *arg3)
 	}
 
 	LOG_INF("HTTP thread terminated");
+	httpc.state = HTTPC_END;
```

Comment:
I think it should go back to HTTPC_INIT here.
Then it would not need any extra state.

But this is OK too.